downstream/modules/platform/ref-hub-variables.adoc

```diff
@@ -38,15 +38,15 @@ Default = `false`.
 {HubNameMain} provides artifacts in `/var/lib/pulp`.
 {ControllerNameStart} automatically backs up the artifacts by default.
 
-You can also set `automationhub_backup_collections = false` and the backup/restore process does not then backup or restore `/var/lib/pulp`.
+You can also set `automationhub_backup_collections to false` and the backup/restore process does not then backup or restore `/var/lib/pulp`.
 
-Default = `true`
+Default = `true`.
 | *`automationhub_collection_download_count`* | _Optional_
 
 Determines whether download count is displayed on the UI.
 
-Default = `false`
-| *`automationhub_collection_seed_repository`* a| When the bundle installer is run, validated content is uploaded to the `validated` repository, and certified content is uploaded to the `rh-certified` repository.
+Default = `false`.
+| *`automationhub_collection_seed_repository`* a| When you run the bundle installer is run, validated content is uploaded to the `validated` repository, and certified content is uploaded to the `rh-certified` repository.
```

Comment:
Should "When you run the bundle installer is run..." 
Instead be "When the bundle installer is run..." ?

Reply:
Good spot! I'm gonna use  with "you run" instead of "is run" as there is a lot of "is" ("is uploaded") that follows and its a bit  samey.